app/apollo/resolvers/channel.js

```diff
@@ -767,6 +801,25 @@ const channelResolvers = {
         } );
         */
 
+        // Create output for graphQL plugins
+        const versionFind = await models.DeployableVersion.find({org_id});
+        const versionObjs = _.map(versionFind, (version)=>{
+          return {
+            name: version.name,
+            uuid: version.uuid,
+          };
+        });
+        const subscriptionFind = await models.Subscription.find({org_id});
+        const subscriptionObjs = _.map(subscriptionFind, (subscription)=>{
+          return {
+            name: subscription.name,
+            uuid: subscription.uuid,
+          };
+        });
+
+        // Allow graphQL plugins to retrieve more information
+        context.pluginContext = {channel: {name: channel.name, uuid: channel.uuid}, versions: versionObjs, subscriptions: subscriptionObjs};
+
```

Comment:
`editChannelVersion` **only** affects a single version record, right?  So the pluginContext shouldn't need channel or subscription details..... it should just pull the name and uuid out of the `version` object retrieved earlier in the function.  No additional queries of 'all versions' or 'all subscriptions' needed.

Reply:
An argument could be made for including channel details (since a version is a 'sub object' of a Channel), but that would also not need any additional queries as the `channel` variable was already retrieved.